language.md

```diff
@@ -135,3 +135,5 @@ As jy Afrikaans kan praat, gaan [hier](afrikaans/kos.md)
 Kama unazungumza kiswahili [ingia hapa](swahili/marshmallow.md)
 
 If you speak Gujarati, go [here](Gujarati/gujarati.md).
+If you learn Telugu, click [here](telugu/hyderabad-telugu/telugu.md).
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.